spec/presenters/guide_presenter_spec.rb

```diff
@@ -28,15 +28,18 @@
       expect(presenter.content_payload).to be_valid_against_schema('service_manual_guide')
     end
 
+    describe "common service manual draft payload" do
+      let(:payload) { presenter.content_payload }
+
+      include_examples "common service manual draft payload"
```

Comment:
I went off `it_behaves_like` because it doesn't always make sense depending on the text. eg. `it_behaves_like "catch http errors"`. Whereas `include_examples` always works because you don't even try to read it like a sentence..

it_behaves_like would read sensibly if it was "a common service manual draft payload" so we could change it.

Generally I find it a losing battle..